server/db/src/main/java/com/walmartlabs/concord/db/migration/SecretsSaltMigrationTask.java

```diff
@@ -0,0 +1,69 @@
+package com.walmartlabs.concord.db.migration;
+
+/*-
+ * *****
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2022 Walmart Inc.
+ * -----
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =====
+ */
+
+import com.walmartlabs.ollie.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+@Named
+@MigrationOrder
+public class SecretsSaltMigrationTask implements MigrationTask {
```

Comment:
maybe replace this task with simple liquibase changelog?

salt we can provide as env variable:
https://docs.liquibase.com/concepts/changelogs/property-substitution.html

Reply:
Yes @brig  , I tried using the env variables, but its not working. seems that feature is not available in open source version. https://www.liquibase.com/blog/liquibase-environment-variables-control-deployments (bottom of the blog).